alignak/alignakobject.py

```diff
@@ -32,13 +32,26 @@ class AlignakObject(object):
     """
 
     properties = {'uuid': StringProp(default='')}
+    macros = {}
 
     def __init__(self, params=None, parsing=True):  # pylint: disable=W0613
 
         if params is None:
             return
+        hasmacro = False
+        if hasattr(self, 'macros'):
+            hasmacro = True
         for key, value in params.iteritems():
-            setattr(self, key, value)
+            if key in ['already_start_escalations', 'tags', 'notified_contacts',
```

Comment:
You should not be based on KEY in AlignakObject. If you want sets just test if key in self.properties and is a SetProp. The list here is only for a few objects.